src/canvas-extractor/canvas_extractor/extract_facade.py

```diff
@@ -77,14 +77,15 @@ def extract_sections(
 
     Returns
     -------
-    Tuple[List[Section], DataFrame]
-        A tuple with the list of Canvas Section objects and the udm_sections dataframe.
+    Tuple[List[Section], DataFrame, List[str]]
+        A tuple with the list of Canvas Section objects, the udm_sections dataframe,
+        and a list of all section ids as strings.
     """
     sections: List[Section] = sectionsApi.request_sections(courses)
     sections_df: DataFrame = sectionsApi.sections_synced_as_df(sections, sync_db)
     udm_sections_df: DataFrame = sectionsMap.map_to_udm_sections(sections_df)
 
-    return (sections, udm_sections_df)
+    return (sections, udm_sections_df, udm_sections_df["SourceSystemIdentifier"].astype("string").tolist())
```

Comment:
To help the new reader grok this more quickly, what do you think about this?
```suggestion
    section_ids = udm_sections_df["SourceSystemIdentifier"].astype("string").tolist()
    return (sections, udm_sections_df, section_ids)
```

Reply:
👍